cilium/ebpf.io-summit
# feat: add new logos to Members

**Describe what changes this pull request brings**
- Added new two logos Tigera & Huawei as Platinum members
- Moved the FutureWei logo to Silver members


**Describe any additional changes**
N/A


**Steps to test**
1. Open [the page](https://deploy-preview-25--ebpf-summit-2021.netlify.app/foundation)
2. Confirm that everything looks and works as expected


**Screenshots**
![image](https://user-images.githubusercontent.com/48465000/144379169-9ded3a40-6434-4de5-85ef-88aa66a0522d.png)




**TODO**
N/A


**References**
N/A
